hyperspy/gui/__init__.py

```diff
@@ -38,7 +38,7 @@ def set_ets_toolkit(toolkit):
 _logger.debug('Current MPL backend: %s', backend)
 if "WX" in backend:
     set_ets_toolkit("wx")
-elif "Qt" in backend:
+elif "Qt4" in backend:
```

Comment:
No, it's not fixing #1559. This fixes using that qt5 backend albeit without the traitsui gui as it does not yet support qt5.

The ipywidgets give us a lot more flexibility when choosing the mpl backend as the ipywidgets are obviously backend independent. Therefore this PR should make hspy usable with e.g. tk or nbagg.